hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3357,3 +3357,105 @@ def hpat_pandas_series_cov_impl(self, other, min_periods=None):
         return numpy.cov(self_arr, other_arr)[0, 1]
 
     return hpat_pandas_series_cov_impl
+
+
+@overload_method(SeriesType, 'pct_change')
+def hpat_pandas_series_pct_change(self, periods=1, fill_method='pad', limit=None, freq=None):
+    """
+    Pandas Series method :meth:`pandas.Series.pct_change` implementation.
+
+    Note: Unsupported mixed numeric and string data
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_series_pct_change
+
+    Parameters
+    -----------
+    self: :obj:`pandas.Series`
+        input series
+    periods: :obj:`int`, default 1
+        Periods to shift for forming percent change.
+    fill_method: :obj:`str`, default 'pad'
+        How to handle NAs before computing percent changes.
+    limit: :obj:`int`, default Nogne
+        The number of consecutive NAs to fill before stopping.
+        *unsupported*
+    freq: :obj: DateOffset, timedelta, or offset alias string, optional
+        Increment to use from time series API (e.g. 'M' or BDay()).
+        *unsupported*
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    ty_checker = TypeChecker('Method pct_change().')
+    ty_checker.check(self, SeriesType)
+
+    if not isinstance(self.data.dtype, types.Number):
+        ty_checker.raise_exc(self.data.dtype, 'number', 'self.data')
+
+    if not isinstance(periods, (types.Integer, types.Omitted)):
```

Comment:
Please add test case with the following implementation:
```
def test_series_pct_change_impl(S):
    return S.pct_change()
```
Perhaps it will be failed on the typing stage.